tests/manual/popup-test.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
```

Comment:
```suggestion
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<meta name="viewport" content="width=device-width, user-scalable=no"/>
```

With meta viewport you get nice mobile friendly scaling => no need to wrap `<h1>`s inside `<div>` to get them styled.